### DISCUSSION TITLE
Imagemagick

### DISCUSSION THREAD

**Discussion by Jcakon** [ImageMagick/ImageMagick#6460] [Help]
### ImageMagick version

7.1.1-12

### Operating system, version and so on

centos

### Description

When I use commands "convert ~/LDBlockShow-1.40/bin/LD_large.svg LD_large.png"
The following situations occur：“convert: no images defined `LD_large.png' @ error/convert.c/ConvertImageCommand/3342.”
What should I do to solve this problem?
Thank you!
Zian Wei

**Comment by snibgo** [comment]
I suggest using `magick` instead of `convert`.

Is that the only message you get? I suspect that is the *last* message, and that other messages were issued.

Please link to your file `LD_large.svg`.

**Comment by Jcakon** [comment]
Here is my code and svg file.
Thank you!








◆_第㈡颗、纽扣
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "ImageMagick/ImageMagick"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年7月4日(星期二) 晚上11:08
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [ImageMagick/ImageMagick] Imagemagick (Discussion #6460)





 
I suggest using magick instead of convert.
 
Is that the only message you get? I suspect that is the last message, and that other messages were issued.
 
Please link to your file LD_large.svg.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***&gt;


从QQ邮箱发来的超大附件

LD.svg (328.91M, 2023年08月03日 23:53 到期)进入下载页面：http://mail.qq.com/cgi-bin/ftnExs_download?t=exs_ftn_download&k=023062300e0912de3d8f712b4066511a414451065104505614520401514b00040c534f5654025a1808545b05075e53075f055a07666063797d1e114601665e&code=90b0ffc5

**Comment by fmw42** [comment]
What SVG renderer do you use?  Inkscape, RSVG delegate or IM internal MSVG/XML?  Add -verbose to your command to find out.  If Inkscape is installed on your system, it will be used in preference to the other two. If you have not installed RSVG, then you are likely using IM internal MSVG/XML, which is the least functional.  I suggest installing Inkscape and test again.
